docs/user/dql/functions.rst

```diff
@@ -1747,20 +1747,21 @@ Description
 >>>>>>>>>>>
 
 Usage: minute(time) returns the minute for time, in the range 0 to 59.
+The `minute_of_hour` function is provided as an alias.
 
 Argument type: STRING/TIME/DATETIME/TIMESTAMP
 
 Return type: INTEGER
 
 Example::
 
-    os> SELECT MINUTE((TIME '01:02:03'))
```

Comment:
why change from TIME to time()?

Reply:
I asked for this, but I recognized later that it is not needed. My mistake.